.gitignore

```diff
@@ -0,0 +1 @@
+/.idea
```

Comment:
Please delete this file from the pull request.

(By the way: you can put locally ignored files in .git/info/exclude, same format as for .gitignore)